digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectValidator.scala

```diff
@@ -580,11 +588,12 @@ object ProjectValidator {
     * 5) If the next road part has differing ely code then there must be a discontinuity code 3 at the end
     *
     * @param project Road address project
-    * @param seq Project links
+    * @param roadProjectLinks Project links
     * @return
     */
-  def checkRoadContinuityCodes(project: RoadAddressProject, seq: Seq[ProjectLink], isRampValidation: Boolean = false): Seq[ValidationErrorDetails] = {
+  def checkRoadContinuityCodes(project: RoadAddressProject, roadProjectLinks: Seq[ProjectLink], isRampValidation: Boolean = false): Seq[ValidationErrorDetails] = {
```

Comment:
roadProjectLinks -> projectLinks

Reply:
There's also "allProjectLinks". I think "projectLinks" is bad because it's kind of upper class for "allProjectLinks" and "roadProjectLinks". Here "roadProjectLinks" contains only project links with one specific road number and "allProjectLinks" contains all links in the project.